platform/view/core/manager/manager.go

```diff
@@ -242,21 +242,18 @@ func (cm *manager) InitiateViewWithIdentity(view view.View, id view.Identity, c
 }
 
 func (cm *manager) InitiateContext(view view.View) (view.Context, error) {
-	return cm.InitiateContextWithIdentity(view, cm.me())
+	return cm.InitiateContextFrom(cm.ctx, view, cm.me(), "")
```

Comment:
We don't need to modify these two methods, right? InitiateContext and InitiateContextWithIdentity

Reply:
I wanted to keep their signatures as they were to not change the API. That's why I pass the default values to the new InitiateContextFrom function (which has an additional parameter for the context).